implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -88,7 +88,7 @@ async fn setup(ctx: Context, c: CreateCommand) -> anyhow::Result<()> {
     let bind = format!("0.0.0.0:{}", c.port);
     tcp.listen(&bind).await?;
 
-    let s = auth::store::mem::Store::new();
+    let s = ockam::identity::authenticated_table::mem::Store::new();
```

Comment:
Help me understand the thought behind this? The authenticated table belongs to the authenticated service and not to one identity.

Reply:
So, I moved this Storage trait to the ockam_identity, because during Identity Secure Channel, participated Identity needs access to the Authenticated Table. Implementations of that trait may be located anywhere, but moving them along is easier because we can use them in Identity tests

Reply:
Also, possible this can be moved down the dependency tree, e.g. to `ockam_core`